config/20.1/make.conf

```
@@ -76,7 +76,7 @@ security_fwknop_UNSET=		IPFW
 security_openconnect_SET=	GNUTLS
 security_openconnect_UNSET=	GSSAPI
 security_openssh-portable_UNSET=HPN
-security_openssl_UNSET=		SSL2 SSL3
+security_openssl_UNSET=		SSL2 SSL3 enable-ec_nistp_64_gcc_128
```

```suggestion
security_openssl_UNSET=		EC SSL2 SSL3
```

Shouldn't this be the proper thing? We unset the option, not the build details :)

---

Well, i still have to learn a lot here ;-)